platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -605,21 +605,21 @@ check_package_manager() {
     not_found=0
     package_managers="apt-get dnf yum dpkg rpm"
     for package in $package_managers; do
-    {
-        res="$(need_cmd $package)"  
-        if [ $? -eq 0 ] ; then
-            not_found=0
-            wrap_debug "Current target platform supports $package package manager"
-            wrap_pass "check_package_manager"
-            if [ $package = "rpm" ] || [ $package = "dpkg" ]; then
-                check_ca_cert
+        {
```

Comment:
it got intended by mistake?

Reply:
I removed the curly braces, what I have seen in shell/bash scripts is that indentation alone is enough.